tech-specs/src/specs/overview.md

```diff
@@ -0,0 +1,20 @@
+# Overview
+
+At high level, Anoma is composed of two main components: distributed ledger and intent gossip with matching system. While they are designed to complement each other, they can be operated separately.
+
+## The ledger
+
+The ledger is a distributed state machine, relying on functionality provided by [Tendermint](https://docs.tendermint.com/master/spec/) such as its BFT consensus algorithm with instant finality, P2P networking capabilities, transaction mempool and more. The ledger state machine is built on top the [ABCI](https://docs.tendermint.com/master/spec/abci/).
+
+For proposer selection, the ledger employs a proof-of-stake system.
```

Comment:
well, yes, but mostly just for validator voting power assignment

Reply:
Good point. I've corrected this, thank you!